### PR TITLE
Sat word update

### PR DESCRIPTION
**Pull request by secondl1ght**
Update all instances of 'sats', 'satoshi' etc. to a unified 'sat' wording as decided in #308. I think I have found them all but I may have missed some. If so just let me know.

### CODE REVIEW

**Comment by bumi** [comment]
super cool! thanks for the PR. 
@MoritzKa can you have a look at the text? what do you think: is it OK to use `sat` everywhere or are some better with `Satoshi`?

**Comment by secondl1ght** [comment]
Yes I changed them all to ```sat``` for now but we may decide some variation might make sense!

File: src/app/screens/LNURLPay.tsx [hunk 1]
```diff
@@ -130,7 +130,7 @@ function LNURLPay(props: Props) {
 
   function renderAmount() {
     if (details.minSendable === details.maxSendable) {
-      return <p>{`${details.minSendable / 1000} Satoshi`}</p>;
+      return <p>{`${details.minSendable / 1000} sat`}</p>;
```
Comment (MoritzKa) [inline src/app/screens/LNURLPay.tsx:133]: not sure where this function is used in the extension. I would keep Satoshi if there is no amount mentioned before.

Comment (bumi) [reply src/app/screens/LNURLPay.tsx:133]: in this case there is an amount `{`${details.minSendable / 1000}` that is basically code that inserts an amount.

File: src/app/screens/LNURLPay.tsx [hunk 2]
```diff
@@ -193,7 +193,7 @@ function LNURLPay(props: Props) {
           {[
             ["Send payment to", details.domain],
             ...formattedMetadata(),
-            ["Amount (Satoshi)", renderAmount()],
+            ["Amount (sat)", renderAmount()],
```
Comment (bumi) [inline src/app/screens/LNURLPay.tsx:196]:
```suggestion
            ["Amount (Satoshi)", renderAmount()],
```

let's keep amount here as of @MoritzKa suggestion

File: src/app/screens/Onboard/TestConnection/index.jsx [hunk 3]
```diff
@@ -113,7 +113,7 @@ export default function TestConnection() {
         className="absolute rounded-lg bg-white w-full max-w-lg"
       >
         <div className="p-5 flex justify-between">
-          <h2 className="text-2xl font-bold">Get some Satoshi</h2>
+          <h2 className="text-2xl font-bold">Get some sat</h2>
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.jsx:116]:
```suggestion
          <h2 className="text-2xl font-bold">Get some Satoshi</h2>
```

File: src/app/screens/Onboard/TestConnection/index.jsx [hunk 4]
```diff
@@ -144,7 +144,7 @@ export default function TestConnection() {
           {faucetLoading ? (
             <Loading />
           ) : (
-            <Button onClick={claimSats} label="Get Sats" primary />
+            <Button onClick={claimSats} label="Get sat" primary />
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.jsx:147]:
```suggestion
            <Button onClick={claimSats} label="Get Satoshi" primary />
```

File: src/app/screens/Onboard/TestConnection/index.jsx [hunk 5]
```diff
@@ -182,7 +182,7 @@ export default function TestConnection() {
                           setShowFaucet(true);
                         }}
                       >
-                        To get started we can send you some Sats...
+                        To get started we can send you some sat...
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.jsx:185]:
```suggestion
                        To get started we can send you some Satoshi...
```

File: src/app/screens/Receive.tsx [hunk 6]
```diff
@@ -119,7 +119,7 @@ function Receive() {
               <div className="mt-1">
                 <Input
                   name="amount"
-                  placeholder="Amount in Satoshi..."
+                  placeholder="Amount in sat..."
```
Comment (bumi) [inline src/app/screens/Receive.tsx:122]:
```suggestion
                  placeholder="Amount in Satoshi..."
```

**Review by bumi — CHANGES_REQUESTED** [review]
Some changes regarding @MoritzKa suggestion.

File: src/app/screens/Home/index.jsx [hunk 7]
```diff
@@ -278,7 +278,7 @@ class Home extends React.Component {
                   this.setState({ loadingSendSats: false });
                 }
               }}
-              label="⚡️ Send Sats ⚡️"
+              label="⚡️ Send sat ⚡️"
```
Comment (bumi) [inline src/app/screens/Home/index.jsx:281]:
```suggestion
              label="⚡️ Send Satoshis ⚡️"
```

File: src/app/screens/Onboard/TestConnection/index.jsx [hunk 8]
```diff
@@ -144,7 +144,7 @@ export default function TestConnection() {
           {faucetLoading ? (
             <Loading />
           ) : (
-            <Button onClick={claimSats} label="Get Sats" primary />
+            <Button onClick={claimSats} label="Get Satoshi" primary />
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.jsx:147]:
```suggestion
            <Button onClick={claimSats} label="Get Satoshis" primary />
```

File: src/app/screens/Onboard/TestConnection/index.jsx [hunk 9]
```diff
@@ -182,7 +182,7 @@ export default function TestConnection() {
                           setShowFaucet(true);
                         }}
                       >
-                        To get started we can send you some Sats...
+                        To get started we can send you some Satoshi...
```
Comment (bumi) [inline src/app/screens/Onboard/TestConnection/index.jsx:185]:
```suggestion
                        To get started we can send you some Satoshis...
```

**Review by bumi — APPROVED** [review]
another round

**Comment by bumi** [comment]
super cool! thanks @secondl1ght

**Comment by secondl1ght** [comment]
Wooo my first merged PR with Alby!! Thanks :)

**Comment by bumi** [comment]
yeah, I am also super excited. great to have you with Alby.

**Pull request merged**